src/main/resources/application.properties

```diff
@@ -1,7 +1,9 @@
 spring.application.name=backstage-plugin-risk-crypto-service
 
-management.server.port=8081
+management.server.port=8083
```

Comment:
Tror porten må være 8081 når det deployes, men så tror jeg også Torbjørn ikke var helt sikker.

Reply:
Ja, jeg er ikke 100% sikker på om vi forvirrer for SKIP her. Jeg satt til 8083 og 8084 lokalt slik at tjenesten ikke skulle bruke de samme portene som backend når vi kjører lokalt, med i SKIP-configen ([skvis-apps](https://github.com/kartverket/skvis-apps/blob/main/applications/cryptoService.libsonnet)) er portene satt til 8080 og 8081